src/common/Image.h

```diff
@@ -28,9 +28,6 @@ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 /*===========================================================================
   Lazy stuff
 ===========================================================================*/
-#define color_t Color /* TODO: search and replace instead ;) */
-#define _Assert(x)
-#define String std::string
 #define CError Exception
```

Comment:
Are you planning to fully remove this define too before merge?

Reply:
Oops :smile: